config/locales/en.yml

```diff
@@ -336,7 +336,9 @@ en:
       rotate_certificate_read_steps: The process for rotating each certificate is slightly different. Read the steps on each page carefully, even if you’ve rotated certificates before.
       MSA: Before you upload your Matching Service Adapter (MSA) %{type} certificate,
       VSP: Before you upload your Verify Service Provider (VSP) %{type} certificate,
-      SP: Before you upload your service provider %{type} certificate,
+      SP: 'Before you upload your service provider %{type} certificate,'
```

Comment:
Should this have a `:` on the end rather than `,`?

Reply:
Also the commas on the lines above (337 and 338). Should `service provider` have a capital S and P?